config/application.rb

```diff
@@ -8,6 +8,7 @@
 
 module BooksApp
   class Application < Rails::Application
+    config.i18n.default_locale = :ja
```

Comment:
コメントありがとうございます。
`config/initializers/locale.rb`の方に一本化しました！

configファイルの優先順位について質問です。
`config/application.rb`、`config/initializers/locale.rb`どちらが優先されるのかふと疑問に思い、実際に試してみました。
結果`config/application.rb`の方が優先度が高かったです。
やはり、Railsに初めから実装されているファイルの方が優先度が高いのでしょうか？
それとも実際に試してみないとわからないケースバイケースなのでしょうか？

１、デフォルトは`:en`表示
<img width="984" alt="スクリーンショット 0005-06-27 16 08 22" src="https://github.com/zamami/fjord-books_app-2023/assets/24470799/86798e9c-67b9-4045-8c29-ba71783ca7e4">

２、`config/initializers/locale.rb`側で`:ja`に設定
<img width="956" alt="スクリーンショット 0005-06-27 16 08 53" src="https://github.com/zamami/fjord-books_app-2023/assets/24470799/ccd56b59-0e13-4c95-a74d-de14ad41233b">

３、`config/application.rb`に`:en`を設定したら`:en`が優先された。

<img width="1021" alt="スクリーンショット 0005-06-27 16 09 19" src="https://github.com/zamami/fjord-books_app-2023/assets/24470799/ae610626-1841-4fdd-aef9-af9d78b4f57d">

Reply:
確認ありがとうございます。
質問する前にRailsガイドの確認怠っていました申し訳ありません💦
> instead of config/application.rb とあるので、あくまでも config/application.rb に設定があるときはそちら、config/application.rb に書かない場合は config/initializers/locale.rb に指定のように書いてもらうと動く、という想定なのかもしれません。

挙動を試してみたことと、こちらのもとに再度Railsガイドを見てみました。
基本的にガイドに記述されている通り、```config/application.rb```が基本ということで理解します。ありがとうございました！